GPU/Common/ShaderId.cpp

```diff
@@ -29,11 +29,13 @@ std::string VertexShaderDesc(const VShaderID &id) {
 		const char *uvprojModes[4] = { "TexProjPos ", "TexProjUV ", "TexProjNNrm ", "TexProjNrm " };
 		desc << uvprojModes[uvprojMode];
 	}
-	const char *uvgModes[4] = { "UV ", "UVMtx ", "UVEnv ", "UVUnk " };
 	int ls0 = id.Bits(VS_BIT_LS0, 2);
 	int ls1 = id.Bits(VS_BIT_LS1, 2);
 
-	if (uvgMode) desc << uvgModes[uvgMode];
+	if (uvgMode) {
+		const char* uvgModes[4] = { "UV ", "UVMtx ", "UVEnv ", "UVUnk " };
```

Comment:
same

Reply:
fixed it here commit: https://github.com/hrydgard/ppsspp/pull/18581/commits/ca94de8d4b1e53b7d596d31e7336ac5133a32a7c